source/config/__init__.py

```diff
@@ -1283,6 +1284,31 @@ def __setitem__(
 		self.manager._markWriteProfileDirty()
 		self._cache[key] = val
 
+		# Alias [documentFormatting][reportFontAttributes] for backwards compatibility.
+		# Todo: Remove in 2025.1.
+		if version_year < 2025and NVDAState._allowDeprecatedAPI():
+			self._link_reportFontAttributes_and_fontAttributeReporting(key, val)
```

Comment:
If keeping backwards compatibility in config is desired, could you please make this more generic? I probably need this for #16823 as well.

#16823 
```suggestion
			self._linkOldValues(key, val)
```